assets/src/apps/authoring/components/Flowchart/rules/create-three-try-workflow.ts

```diff
@@ -122,7 +123,9 @@ export const generateMaxTryWorkflow = (
         incorrect.destinationId,
         false,
         40,
-        options.threeTimesFeedback,
+        extraOptions.maxAttempt == '1' && incorrect.feedback != DEFAULT_INCORRECT_FEEDBACK
```

Comment:
If the author sets the attempts to 1, the max attempt feedback should display on the first incorrect attempt unless the author has provided custom incorrect feedback. In such a case, the custom feedback should be displayed instead.